From: Arnaud Ferraris <arnaud.ferraris@collabora.com>
Date: Mon, 17 Jul 2023 18:58:30 +0200
Subject: arm64: dts: rk3399-pinephone-pro: Fix CSI clock frequency

`SCLK_CIF_OUT` cannot generate precise 19.2MHz. However, the current
`imx258` driver also accepts 24MHz, so let's use that frequency as it'll
be more accurately generated.
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index 4b27117..664c3ab 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -852,7 +852,7 @@ &i2c1 {
 
 	assigned-clocks = <&cru SCLK_CIF_OUT &cru SCLK_CIF_OUT_SRC>;
 	assigned-clock-parents = <&cru SCLK_CIF_OUT_SRC &cru PLL_GPLL>;
-	assigned-clock-rates = <19200000 0>;
+	assigned-clock-rates = <24000000 0>;
 
 	wcam: camera@1a {
 		compatible = "sony,imx258";
